### PR TITLE
[SDK] Add flow diagram for Orchestrator

### PR DESCRIPTION
**Pull request by praysml**
Add flow diagram for module:
- [x] General Orchestrator
- [x] Flow Executor
- [x] LLM
- [x] Search
- [x] Prompt

### CODE REVIEW

**Comment by kevin-yauris** [comment]
All items on the list have been checked. Is this ready to be reviewed? @praysml @prawiraelang @MuvazanaGDPLabs

**Comment by prawiraelang** [comment]
> All items on the list have been checked. Is this ready to be reviewed? @praysml @prawiraelang @MuvazanaGDPLabs

There is another PR in #19 to add/adjust some explanation in the Gitbook
@praysml and @MuvazanaGDPLabs will review PR #19 first and then we will merge PR #19 to this PR #16 

After that, Kak Yauris and others can review PR #16

**Comment by prawiraelang** [comment]
> > All items on the list have been checked. Is this ready to be reviewed? @praysml @prawiraelang @MuvazanaGDPLabs
> 
> There is another PR in #19 to add/adjust some explanation in the Gitbook @praysml and @MuvazanaGDPLabs will review PR #19 first and then we will merge PR #19 to this PR #16
> 
> After that, Kak Yauris and others can review PR #16

PR #19 has been merged; therefore, PR #16 is now ready for review.

File: src/app/internal-sdk-inference-orchestrator/page.mdx [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+relevant data, and utilizes **Prompt** to format the user's question into a structured request. Finally, a crafted response is 
+generated and presented back to the user. 
+
+You can refer to [Installation](internal-sdk#installation) section to install the **Inference Orchestrator**.
 
 List of modules:
 
-1. [Flow Executor](/internal-sdk-inference-orchestrator/prompt)
-2. [Use Case Handler](/internal-sdk-inference-orchestrator/search)
+1. [Flow Executor](/internal-sdk-inference-orchestrator/flow-executor)
```
Comment (timotiusnc) [inline src/app/internal-sdk-inference-orchestrator/page.mdx:36]: Make the order the same as the one in `Navigation.tsx` (or vice versa)

Comment (prawiraelang) [reply src/app/internal-sdk-inference-orchestrator/page.mdx:36]: Updated. I have adjusted the order

**Review by timotiusnc — APPROVED** [review]

**Pull request merged**